### DISCUSSION TITLE
11/13 (水) 23:00~ FE MTG

### DISCUSSION THREAD

**Discussion by penguin2** [42-pong/42-pong#58] [Meetings]
- SPA の勉強
- 次回タスク

**Comment by weijuan82113** [comment]
## 次回タスク
### HTML、CSS、JSのリンカー、フォーマッターについて調査
- Linter：
    - HTML：HTMLHint
    - JavaScript：ESLint
    - CSS：Stylelint
- Formatter：
    - HTML：Prettier
    - JavaScript：Prettier
    - CSS：Prettier、PostCSS
